src/worldcereal/job.py

```diff
@@ -238,7 +103,12 @@ def generate_map(
         raise ValueError(f"Format {format} not supported.")
 
     # Make a connection to the OpenEO backend
-    connection = BACKEND_CONNECTIONS[backend_context.backend]()
+    if backend_context.backend == Backend.CDSE:
+        connection = openeo.connect(
+            "https://openeo.creo.vito.be/openeo/"
```

Comment:
should we add a comment here why we go this way for the time being or not needed?

Reply:
I wouldn't publicly comment "We do this to avoid the firewall" haha

Reply:
With this comment, you just did 😆